### PR TITLE
BAH-3951 | Fix. Default Observation Forms, Dosing Units, and Bacteriology Sample Type Dropdown

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: openmrs/apps/clinical/extension.json [hunk 1]
```diff
@@ -174,29 +174,19 @@
     "extensionPointId": "org.bahmni.clinical.conceptSetGroup.observations",
     "type": "config",
     "extensionParams": {
-      "conceptName": "Obstetrics"
+      "formName": "Obstetrics and Gynaecology"
```
Comment (mohan-13) [inline openmrs/apps/clinical/extension.json:177]: @SanoferSameera do we have this form configured in our metadata ?

File: masterdata/configuration/concepts/drugForms.csv [hunk 2]
```diff
@@ -2,4 +2,6 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Descript
 9bb0795c-4ff0-0305-1990-000000000034,,,Injection,,,,,,Misc,N/A,,,,,,,,,,,
 9bb0795c-4ff0-0305-1990-000000000035,,,Drops,,,,,,Misc,N/A,,,,,,,,,,,
 9bb0795c-4ff0-0305-1990-000000000036,,,Jelly,,,,,,Misc,N/A,,,,,,,,,,,
-9bb0795c-4ff0-0305-1990-000000000037,,,Pessary,,,,,,Misc,N/A,,,,,,,,,,,
\ No newline at end of file
+9bb0795c-4ff0-0305-1990-000000000037,,,Pessary,,,,,,Misc,N/A,,,,,,,,,,,
+e466569c-1e90-11ef-8cfd-0242ac120003,,,Tablet(s),tablet,,,,,,Misc,N/A,,,,,,,,,,,
```
Comment (mohan-13) [inline masterdata/configuration/concepts/drugForms.csv]: There are existing concept named as Tablet, Capsule. Are we renaming the same or these are new concepts ?

File: openmrs/apps/clinical/extension.json (hunk 1 above)
Comment (SanoferSameera) [reply openmrs/apps/clinical/extension.json:177]: Yes. here it is https://github.com/Bahmni/standard-config/blob/master/masterdata/configuration/bahmniforms/obstetrics_and_gynaecology.json

File: masterdata/configuration/concepts/drugForms.csv (hunk 2 above)
Comment (SanoferSameera) [reply masterdata/configuration/concepts/drugForms.csv]: Yes, I'm renaming the already existing one by using its same uuid. https://ciel.standard.mybahmni.in/openmrs/dictionary/concept.htm?conceptId=64

Comment (mohan-13) [reply masterdata/configuration/concepts/drugForms.csv]: @SanoferSameera where is this UUID coming from ? Do we have it fixed somewhere in `bahmni-core` ? Can you validate the source of creation of this concept ?

cc. @angshu @gokultw

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**